### ISSUE TITLE
Gridslist in Torch UI is super hard to deal with (always jumps back to top and reloads)

### ISSUE THREAD

**Issue by LordTylus** [TorchAPI/Torch#329]
- [x] Verify that this issue is related to Torch and not a Torch plugin or the vanilla game
- [x] Ensure that the issue is reproducible for testing (provide a link to a test world if necessary)
- [ ] Is this a suggestion?

Torch Version: 1.3.1.64 SE Version: 1.191.106

### Expected Behavior
When I am in the Tree with all Entities on the Server and klick on a specific grid to either stop it, delete it, or change settings. I can do that without problem,

Select the Grid I want, 
check what blocks it has, 
change settings to it

### Observed Behavior
I often dont even get to select the grid, when scrolling down some seemingly random reflesh happens scrolling all the way up again.

Selected Gids will be unselected again

Grids Children will not be expanded anymore.

Its really frustrating to work with this entity list

### Steps to Reproduce
Basically just start your server, have some people play on it and try to check on grids in the entity list. especially grids that are on the bottom of the list.
